### PR TITLE
test(e2e): introduce cypress e2e testing

### PR DESCRIPTION
**Pull request by jeffchew**
### Related Ticket(s)

No related issue

### Description

After our website outage, this will hopefully introduce better checks to ensure that we are notified of any rendering issues on the website. This will run e2e tests (homepage for now) on PRs, and also will run on a cron ever 2 hours on the prod environment.

### Changelog

**New**

- Cypress test harness
- Workflow to run Cypress e2e tests on pull requests
- Workflow to run e2e tests on the prod environment every 2 hours

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,23 +42,19 @@
...
-    "prettier": "^2.3.2"
-  },
-  "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "prettier --write"
-    ],
-    "*.{css,scss,json,html,yaml,md,mdx}": [
-      "prettier --write"
-    ]
```
Comment (emyarod) [inline package.json:58]: is this lint-staged deletion intentional?

**Review by emyarod — COMMENTED** [review]
it looks like there are some lockfile conflicts

File: package.json (hunk 1 above)
Comment (jeffchew) [reply package.json:58]: yes, I moved it to `.lintstagerc` to be consistent with the other config files at the root of the project.

**Comment by jeffchew** [comment]
> it looks like there are some lockfile conflicts

Yup I noticed that, I'm going a reset locally and will push up the changes!

**Review by emyarod — APPROVED** [review]
looks good to me!

**Pull request merged**